exocom-mock-javascript/bin/spec

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 set -e
 
-node_modules/o-tools-livescript/bin/build
+build
```

Comment:
I think this should be `bin/build`, since it will be called from the `exocom-mock-javascript` directory.